runtime/query.go

```diff
@@ -43,13 +43,16 @@ func populateFieldValueFromPath(msg proto.Message, fieldPath []string, values []
 	var props *proto.Properties
 	m = m.Elem()
 	for i, fieldName := range fieldPath {
-		isLast := i == len(fieldPath)-1
+		var isLast = i == len(fieldPath)-1
```

Comment:
Amended commit force-pushed. Didn't realize GitHub supported that now. (Has it always?)